dev/README_RELEASE_PROVIDER_PACKAGES.md

```diff
@@ -1233,37 +1304,6 @@ If you had this issue you will need also to make adjustments in the next step to
 This is simply by removing the relevant files locally.
 
 
-## Publish the packages to PyPI
-
-By that time the packages should be in your dist folder.
-
-```shell script
-cd ${AIRFLOW_REPO_ROOT}
-git checkout <ONE_OF_THE_RC_TAGS_FOR_ONE_OF_THE_RELEASED_PROVIDERS>
-```
-
-example `git checkout providers-amazon/7.0.0rc2`
-
-Note you probably will see message `You are in 'detached HEAD' state.`
-This is expected, the RC tag is most likely behind the main branch.
-
-* Verify the artifacts that would be uploaded:
-
-```shell script
-twine check ${AIRFLOW_REPO_ROOT}/dist/*.whl ${AIRFLOW_REPO_ROOT}/dist/*.tar.gz
-```
-
-* Upload the package to PyPi:
-
-```shell script
-twine upload -r pypi ${AIRFLOW_REPO_ROOT}/dist/*.whl ${AIRFLOW_REPO_ROOT}/dist/*.tar.gz
-```
-
-* Verify that the packages are available under the links printed.
-
-Copy links to updated packages, sort it alphabetically and save it on the side. You will need it for the announcement message.
-
-* Again, confirm that the packages are available under the links printed.
```

Comment:
I am not sure why this is removed.
Or at least why we are changing the order of the steps in the release. I find it confusing.

Reply:
It's not removed - it's moved up. And we are changing the order because we want to also publish the `PyPI` packages in SVN. So far we have not been doing it - because we did not need them in SVN (and it was a bit problematic as well because RC `PyPI` packages never went through our SVN and were never available there nor available in the history.

With "publish" workflow we want to make sure that packages are committed to SVN (including the PyPI packages) - that's why we are moving the "prepare" step before.

And since we eventually want to replace the "publish" step from manual to workflow, it makes sense to put the "manual" step next to the "workflow" test. 

I think it could be also left below, but I found it cleaner if we have "publish" steps manual/workflow next to each other.

Reply:
Let me know @eladkal if that explains it :)

Reply:
I will make the release according to instructions in this PR, that way we can change it on the fly then merge the final version that actually works. It will be easier to write questions/comments on the fly when the PR is up and change it as we progress

Reply:
cool :) Sounds like a plan!